### PR TITLE
Update globalization structure

### PR DESCRIPTION
**Pull request by 0candy**
Follow the new approach that's supported by strong-globalize since https://github.com/strongloop/strong-globalize/pull/92.

### CODE REVIEW

**Comment by 0candy** [comment]
@bajtos PTAL

File: common/models/change.js [hunk 1]
```diff
@@ -7,9 +7,7 @@
  * Module Dependencies.
  */
 
-var g = require('strong-globalize')();
-
-var PersistedModel = require('../../lib/loopback').PersistedModel;
```
Comment (bajtos) [inline common/models/change.js]: I think you have accidentally removed reference to `PersistedModel`. PTAL.

File: lib/globalize.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+var path = require('path');
+var SG = require('strong-globalize');
+
+SG.SetRootDir(path.join(__dirname, '..'), { autonomousMsgLoading: 'all' });
+module.exports = SG();
```
Comment (bajtos) [inline lib/globalize.js]: No EOL at EOF, please fix. I thought `eslint` was configured to catch this, could you PTAL?

**Comment by 0candy** [comment]
@bajtos Please review

**Comment by bajtos** [comment]
> loopback/node=0.10,os=windows — Failed! (640f3a8)

This failure is caused beuse PhantomJS cannot be started, I think it's ok to ignore. (see https://github.com/strongloop-internal/scrum-loopback/issues/1052)

**Comment by bajtos** [comment]
I squashed the commits myself and landed.

**Pull request merged**